src/components/RecipeCard/RecipeCard.js

```diff
@@ -1,11 +1,30 @@
-
+import './RecipeCard.css';
+import { allRecipes, individualUserData, allNPOs } from '../../mockData.js'
+import {Link} from 'react-router-dom'
 
 function RecipeCard() {
+
   return (
     <div className="RecipeCard">
-      <header className="RecipeCard-header">
-        <h1> Recipe Card </h1>
-      </header>
+      <section className='left-section'>
+        <div className='leftHeader'>
+          <h1 className='recipe-name'>{allRecipes[0].title}</h1>
```

Comment:
These will need to be refactored to props so we can dynamically render the cards and not just hard code from mock data, but this is looking like a good start!